advanced/sr_gui_cyberglove_calibrator/src/sr_gui_cyberglove_calibrator/cyberglove_calibrator.py

```diff
@@ -320,32 +323,29 @@ def __init__(self, context):
         self.step_selector = StepSelector(self._widget, self.calibrer)
         sublayout.addWidget(self.step_selector)
 
-        btn_frame = QtGui.QFrame()
-        btn_layout = QtGui.QVBoxLayout()
+        btn_frame = QtWidgets.QFrame()
+        btn_layout = QtWidgets.QVBoxLayout()
         btn_layout.setSpacing(25)
-        btn_calibrate = QtGui.QPushButton()
+        btn_calibrate = QtWidgets.QPushButton()
         btn_calibrate.setText("Calibrate")
         btn_calibrate.setToolTip("Calibrate the current selected step")
         btn_calibrate.setIcon(
             QtGui.QIcon(rootPath + '/images/icons/calibrate.png'))
         btn_layout.addWidget(btn_calibrate)
-        btn_frame.connect(btn_calibrate, QtCore.SIGNAL(
-            'clicked()'), self.calibrate_current_step)
-        self.btn_save = QtGui.QPushButton()
+        btn_frame.btn_calibrate.clicked.connect(self.calibrate_current_step)
```

Comment:
btn_frame gets the layout containing the btn_calibrated added later, so I wonder if this can work (similar for btn_save and btn_load)